Several RoxAPI spesific tests fails after last weeks updates

Testing in RMS' python environment, I get e.g.

`FAILED tests/test_roxarapi/test_roxarapi_reek.py::test_rox_get_gridproperty - FileNotFoundError: PORO`

There more tests failing also

--- a/src/xtgeo/grid3d/grid_property.py
+++ b/src/xtgeo/grid3d/grid_property.py
@@ -183,7 +183,7 @@ def gridproperty_from_roxar(
 
     """
     return GridProperty._read_roxar(
-        pname,
+        project,
         gname,
         pname,
         realisation=realisation,

